src/net.cpp

```diff
@@ -2262,8 +2262,10 @@ void CConnman::SetNetworkActive(bool active)
     uiInterface.NotifyNetworkActiveChanged(fNetworkActive);
 }
 
-CConnman::CConnman(uint64_t nSeed0In, uint64_t nSeed1In) : nSeed0(nSeed0In), nSeed1(nSeed1In)
+CConnman::CConnman(uint64_t nSeed0In, uint64_t nSeed1In, bool network_active)
+    : nSeed0(nSeed0In), nSeed1(nSeed1In)
 {
+    SetNetworkActive(network_active);
```

Comment:
I don't think it makes any difference in practice in this case, but isn't the general convention to call functions like this on the object only after `Init`?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19473#issuecomment-662666625).